docs/build/configure/_index.md

```diff
@@ -13,32 +13,33 @@ aliases:
 
 Before you can program a smart machine, you must configure it.
 
-A machine's configuration tells the code running the machine what _{{< glossary_tooltip term_id="resource" text="resources" >}}_ (hardware _{{< glossary_tooltip term_id="component" text="components" >}}_ and software _{{< glossary_tooltip term_id="service" text="services" >}}_) it has access to, as well as any relevant parameters for those resources.
+A machine's configuration defines the _{{< glossary_tooltip term_id="resource" text="resources" >}}_ (hardware and software services) it has access to, as well as any relevant parameters for those resources.
+You can configure the following resources:
+
+- [Components](/build/configure/#add-a-component): _{{< glossary_tooltip term_id="component" text="Components" >}}_ are the hardware of your machine.
+- [Services](/build/configure/#add-a-service): _{{< glossary_tooltip term_id="service" text="Services" >}}_ are the software that runs on your machine.
+- [Processes](/build/configure/#add-a-process): Processes automatically run specified scripts when the machine boots.
+- [Local Modules](/build/configure/#add-a-local-module): {{< glossary_tooltip term_id="module" text="Modules" >}} provide {{< glossary_tooltip term_id="modular-resource" text="modular resources" >}}, which are a way to add resource types or models that are not built into Viam.
```

Comment:
```suggestion
- [Local modules](/build/configure/#add-a-local-module): {{< glossary_tooltip term_id="module" text="Modules" >}} provide {{< glossary_tooltip term_id="modular-resource" text="modular resources" >}}, which are a way to add resource types or models that are not built into Viam.
```
[nit] consistency with capitalization of remote parts (or change that one)

Reply:
I worry that calling out local modules specifically could be confusing since they provide components and services just like registry modules do, so maybe this should just be modules in general.